pytensor/tensor/blockwise.py

```diff
@@ -340,10 +332,12 @@ def _check_runtime_broadcast(self, node, inputs):
                 )
 
     def perform(self, node, inputs, output_storage):
-        gufunc = self._gufunc
+        gufunc = getattr(node.tag, "gufunc", None)
 
         if gufunc is None:
-            gufunc = self._create_gufunc(node)
+            # Cache it once per node
+            self._create_gufunc(node)
+            gufunc = node.tag.gufunc
```

Comment:
I think this line is causing tests to fail. If we get here, this by definition doesn't exist right?

Reply:
self.create_gufunc should add it

Reply:
Forgot to add it when there's a gufunc_spec